templates/default/less/variables.less

```diff
@@ -340,3 +340,16 @@
 //##
 @il-counter-novelty-bg: @brand-warning;
 @il-counter-novelty-margin-top: -5px;
+
+//== Popover
+//
+//##
+@il-popover-min-width: 50px;
```

Comment:
Please add a description for each variable. See: [Less Guideline](https://github.com/ILIAS-eLearning/ILIAS/tree/trunk/templates#naming)